connectors/luoo.js

```diff
@@ -1,9 +1,109 @@
 'use strict';
+(() => {
+	let setupConnector = () => {
```

Comment:
Could you use function declaration instead of function expression?

Reply:
Okay